### ISSUE TITLE
Check answers

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsuk-service-manual-backlog#313]
## What

A pattern to let users check their answers before submitting information to a service.

## Anything else

[GOV.UK Design System's check answers pattern](https://design-system.service.gov.uk/patterns/check-answers/)

**Comment by anth-NHS** [comment]
The pattern already exists, it's just under a different name to GOV ('Summary list') https://service-manual.nhs.uk/design-system/components/summary-list

**Comment by frankieroberto** [comment]
Here’s a screenshot of how we’ve used the pattern with the NHS Record a vaccination service:

![Image](https://github.com/user-attachments/assets/2b7d4964-dc96-4bbc-924c-6ccefc1e6231)

Things to note:

* the top section with the patient details does not have ‘change’ links, as this information comes from the GP record via the Patient Demographic Service (PDS) and can’t be edited
* we’ve used shorter labels than are on the pages themselves, eg ‘Vaccinator’ rather than ‘Who is the vaccinator?’
* the vaccine (COVID-19) and vaccine product (Comirnaty 30 JN.1) are asked on the same page, so have a single change link and are both covered by the label ‘Vaccine’ as the 2 fields are closely associated
* we also add an optional note field on this page

**Comment by paulrobertlloyd** [comment]
Here’s a screenshot of how we’ve used the pattern in the _Manage vaccinations in schools_ (and parent-facing _Give or refuse consent for vaccinations_) services:

![Screenshot of check and confirm page in Manage vaccinations in schools.](https://github.com/user-attachments/assets/96fe9ef9-c7f8-450c-acd1-690f6aa59b6b)

![Screenshot of check and confirm page in Give or refuse consent for vaccinations.](https://github.com/user-attachments/assets/f60a0865-2443-4bcf-992a-913bebdf17b9)

- We combine the summary list component with the card component; this is especially useful when wanting to group answers to separate sections (there is no [summary card component](https://design-system.service.gov.uk/components/summary-list/#summary-in-a-card-with-a-title-and-actions-summary-list-example) in the NHS design system, which might address this need)
- We use a linearised variation of the summary list component for health answers because both the key and value use longer strings

**Comment by sarawilcox** [comment]
Hi @frankieroberto (and @paulrobertlloyd and @anandamaryon1 ) , on GOV it says: 

![Image](https://github.com/user-attachments/assets/3d612822-aa12-46cf-8d1d-cac043c299d1)

On the draft in https://github.com/nhsuk/nhsuk-service-manual/pull/2141, it says:

![Image](https://github.com/user-attachments/assets/599d738c-98d7-424f-8bf0-3eae46623469)

I'm not keen on that legal bit "By submitting this notification/application". Do you think we need to include it? When we include things in a pattern, people tend to copy it when it may not be necessary. Some examples just have a button at the bottom of the page under the last Change link.

In the [GOV design system button component](https://design-system.service.gov.uk/components/button/), there's a bit of info about writing button copy. It includes examples:
- "Confirm and send" on a [Check answers page](https://design-system.service.gov.uk/patterns/check-answers/) that does not have any legal content a user must agree to
- "Accept and send" on a [Check answers page](https://design-system.service.gov.uk/patterns/check-answers/) that has legal content a user must agree to

Should we use "Confirm and send" on our example?

**Comment by anandamaryon1** [comment]
@sarawilcox 
> Should we use "Confirm and send" on our example?

Yes I think in our example – a member of the public sending an application – 'Confirm and send' makes sense. But maybe somewhere we should explain that for staff systems, or depending on your use-case, 'Confirm and save' may be more appropriate. Perhaps just a new sentence in the guidance page, and/or a mention on the button's component page's button text examples?

**Comment by sarawilcox** [comment]
I'm tempted to add it to this Check answers page. People may not visit the button guidance.

**Comment by frankieroberto** [comment]
Feel free to edit or remove that bit! I copied it from the pattern in the NHS prototype kit, but I’m not sure where that came from.

(I’m on holiday this week so feel free to either make any commits directly to the branch or wait for me to be back next week)
